### PR TITLE
add credentials same-origin when fetching _devPagesManifest.json

### PR DESCRIPTION
**Pull request by JorgePasco1**
This should do for #17274

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| buildDuration | 13.6s | 13.3s | -250ms |
| nodeModulesSize | 63.4 MB | 63.4 MB | ⚠️ +250 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.457 | 2.443 | -0.01 |
| / avg req/sec | 1017.59 | 1023.26 | +5.67 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.323 | 1.324 | 0 |
| /error-in-render avg req/sec | 1889.31 | 1888.89 | ⚠️ -0.42 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..977d.js gzip | 11 kB | 11 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-6f0de25..2754.js gzip | 7.21 kB | 7.21 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58 kB | 58 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.87 kB | 6.87 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-9ee6725..dule.js gzip | 6.27 kB | 6.27 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 52.9 kB | 52.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-9a0b9e1..b37e.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-ed1b0..8fbd.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-89731c..c609.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-537beac..0f9a.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-02f45f5..dule.js gzip | 1.28 kB | 1.28 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 653 B | 653 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| index.html gzip | 1 kB | 1 kB | ✓ |
| link.html gzip | 1.01 kB | 1.01 kB | ✓ |
| withRouter.html gzip | 994 B | 994 B | ✓ |
| Overall change | 3 kB | 3 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-8f151396166789e5f006.module.js"
+      href="/_next/static/chunks/main-6d991f710c306a120403.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-e85a0b19170a6ee0345b.js"
+      src="/_next/static/chunks/main-36eb32eed8d7cde906b3.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-8f151396166789e5f006.module.js"
+      src="/_next/static/chunks/main-6d991f710c306a120403.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-8f151396166789e5f006.module.js"
+      href="/_next/static/chunks/main-6d991f710c306a120403.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -90,13 +90,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-e85a0b19170a6ee0345b.js"
+      src="/_next/static/chunks/main-36eb32eed8d7cde906b3.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-8f151396166789e5f006.module.js"
+      src="/_next/static/chunks/main-6d991f710c306a120403.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-8f151396166789e5f006.module.js"
+      href="/_next/static/chunks/main-6d991f710c306a120403.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-e85a0b19170a6ee0345b.js"
+      src="/_next/static/chunks/main-36eb32eed8d7cde906b3.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-8f151396166789e5f006.module.js"
+      src="/_next/static/chunks/main-6d991f710c306a120403.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| buildDuration | 14.9s | 14.6s | -241ms |
| nodeModulesSize | 63.4 MB | 63.4 MB | ⚠️ +250 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..977d.js gzip | 11 kB | 11 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-6f0de25..2754.js gzip | 7.21 kB | N/A | N/A |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| main-6bb7d86..4ad2.js gzip | N/A | 7.21 kB | N/A |
| Overall change | 58 kB | 58 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.87 kB | 6.87 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-9ee6725..dule.js gzip | 6.27 kB | N/A | N/A |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| main-a6034da..dule.js gzip | N/A | 6.27 kB | N/A |
| Overall change | 52.9 kB | 52.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-9a0b9e1..b37e.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-ed1b0..8fbd.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-89731c..c609.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-537beac..0f9a.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-02f45f5..dule.js gzip | 1.28 kB | 1.28 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 653 B | 653 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _error.js | 1.05 MB | 1.05 MB | ✓ |
| 404.html | 4.34 kB | 4.34 kB | ✓ |
| hooks.html | 3.92 kB | 3.92 kB | ✓ |
| index.js | 1.05 MB | 1.05 MB | ✓ |
| link.js | 1.1 MB | 1.1 MB | ✓ |
| routerDirect.js | 1.09 MB | 1.09 MB | ✓ |
| withRouter.js | 1.09 MB | 1.09 MB | ✓ |
| Overall change | 5.41 MB | 5.41 MB | ✓ |

</details>

</details>
Commit: cc441275b0e6d1f0d11098a15daa2d189e4cd478

File: test/integration/src-dir-support/next.config.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
 module.exports = {
-  experimental: {
-    modern: true,
-  },
+  target: 'serverless',
```
Comment (timneutkens) [inline test/integration/src-dir-support/next.config.js:2]: These changes seem unwanted.

File: test/integration/typescript-ignore-errors/next.config.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = { typescript: { ignoreBuildErrors: false } }
```
Comment (timneutkens) [inline test/integration/typescript-ignore-errors/next.config.js:1]: Same as https://github.com/vercel/next.js/pull/17702/files#r501564532

**Comment by JorgePasco1** [comment]
Corrected that!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| buildDuration | 12.2s | 11.8s | -340ms |
| nodeModulesSize | 63.4 MB | 63.4 MB | ⚠️ +250 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.229 | 2.216 | -0.01 |
| / avg req/sec | 1121.73 | 1128.35 | +6.62 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.295 | 1.248 | -0.05 |
| /error-in-render avg req/sec | 1930.29 | 2003.9 | +73.61 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..9b19.js gzip | 11.1 kB | 11.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-310f69a..b28a.js gzip | 7.23 kB | 7.23 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58 kB | 58 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.9 kB | 6.9 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-fbbf49e..dule.js gzip | 6.29 kB | 6.29 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 52.9 kB | 52.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-9a0b9e1..b37e.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-ed1b0..8fbd.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-89731c..c609.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-409b283..e3ab.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.73 kB | 7.73 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-92d3016..dule.js gzip | 1.28 kB | 1.28 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| _buildManife..dule.js gzip | 329 B | 329 B | ✓ |
| Overall change | 652 B | 652 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| index.html gzip | 1 kB | 1 kB | ✓ |
| link.html gzip | 1.01 kB | 1.01 kB | -1 B |
| withRouter.html gzip | 996 B | 995 B | -1 B |
| Overall change | 3.01 kB | 3.01 kB | -2 B |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-500620fa408ba2b80f1c.module.js"
+      href="/_next/static/chunks/main-e95124ae1a8475c663c6.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-53ff02d086cacb9abd03.js"
+      src="/_next/static/chunks/main-92e7f129bcc5df22823f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-500620fa408ba2b80f1c.module.js"
+      src="/_next/static/chunks/main-e95124ae1a8475c663c6.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-500620fa408ba2b80f1c.module.js"
+      href="/_next/static/chunks/main-e95124ae1a8475c663c6.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -90,13 +90,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-53ff02d086cacb9abd03.js"
+      src="/_next/static/chunks/main-92e7f129bcc5df22823f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-500620fa408ba2b80f1c.module.js"
+      src="/_next/static/chunks/main-e95124ae1a8475c663c6.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-500620fa408ba2b80f1c.module.js"
+      href="/_next/static/chunks/main-e95124ae1a8475c663c6.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-53ff02d086cacb9abd03.js"
+      src="/_next/static/chunks/main-92e7f129bcc5df22823f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-500620fa408ba2b80f1c.module.js"
+      src="/_next/static/chunks/main-e95124ae1a8475c663c6.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| buildDuration | 13.7s | 13.5s | -231ms |
| nodeModulesSize | 63.4 MB | 63.4 MB | ⚠️ +250 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..9b19.js gzip | 11.1 kB | 11.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-310f69a..b28a.js gzip | 7.23 kB | N/A | N/A |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| main-55dc556..8fd3.js gzip | N/A | 7.23 kB | N/A |
| Overall change | 58 kB | 58 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.9 kB | 6.9 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-fbbf49e..dule.js gzip | 6.29 kB | N/A | N/A |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| main-8541fb3..dule.js gzip | N/A | 6.29 kB | N/A |
| Overall change | 52.9 kB | 52.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-9a0b9e1..b37e.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-ed1b0..8fbd.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-89731c..c609.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-409b283..e3ab.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.73 kB | 7.73 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-92d3016..dule.js gzip | 1.28 kB | 1.28 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| _buildManife..dule.js gzip | 329 B | 329 B | ✓ |
| Overall change | 652 B | 652 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | JorgePasco1/next.js same-origin-for-manifest | Change |
| - | - | - | - |
| _error.js | 1.05 MB | 1.05 MB | ✓ |
| 404.html | 4.34 kB | 4.34 kB | ✓ |
| hooks.html | 3.92 kB | 3.92 kB | ✓ |
| index.js | 1.05 MB | 1.05 MB | ✓ |
| link.js | 1.1 MB | 1.1 MB | ✓ |
| routerDirect.js | 1.09 MB | 1.09 MB | ✓ |
| withRouter.js | 1.09 MB | 1.09 MB | ✓ |
| Overall change | 5.41 MB | 5.41 MB | ✓ |

</details>

</details>
Commit: 08b5beb54f593d98487f63b5860945eab53b31c3

**Comment by timneutkens** [comment]
Can you add an integration test for this behavior change in test/integration?

**Pull request closed without merging**